README.md

```diff
@@ -17,9 +27,35 @@ dependencyOverrides += "com.puppycrawl.tools" % "checkstyle" % "7.7"
 Then to use a particular config reference it's URL in your config location:
 ```
 checkstyleConfigLocation :=
-  CheckStyleConfigLocation.URL("https://raw.githubusercontent.com/SocialFinance/sofi-checks/master/checks/checkstyle-config.xml")
+  CheckStyleConfigLocation.URL("https://raw.githubusercontent.com/SocialFinance/sofi-checks/master/checks/java/checkstyle-config.xml")
+```
+To have checkstyle run as part of the compile and fail on style errors add the following to the build.sbt
+```
+checkstyleSeverityLevel := Some(CheckstyleSeverityLevel.Error)
+(checkstyle in Compile) <<= (checkstyle in Compile) dependsOn (compile in Compile)
+```
+
+### Scalastyle
+
+To check coding style for scala use the [scalastyle plugin](http://www.scalastyle.org/)
+Add the following to the project/plugins.sbt file
+```
+//Scalastyle plugin
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+```
+
+And the configuration goes in the build.sbt
+```
+(scalastyleConfigUrl in Compile) := Some(url("https://raw.githubusercontent.com/SocialFinance/sofi-checks/master/scala/checkstyle/scalastyle-config.xml"))
```

Comment:
I think this URL should be something like `https://raw.githubusercontent.com/SocialFinance/sofi-checks/master/checks/scala/scalastyle-config.xml` (but obviously it won't work until it's merged)